README.md

```diff
@@ -1,13 +1,16 @@
-# puppeteer-cli
+# puppeteer-v-cli
 
 A general command-line wrapper for puppeteer. Currently only supports one command—`print`—to render a local/or remote HTML file to PDF. Aims to be a easy replacement for the deprecated wkhtmltopdf.
 
+It is copycat of `puppeteer-cli`. Just support viewport option. (when original app support this feature, it will be deprecated.)
+
 ## Usage
 
 ```bash
-puppeteer print <input> <output>
+puppeteer-v print <input> <output>
+
+Print an HTML file or URL to PDF. If viewport-width > 0 and viewport-height > 0, then set viewport size.
```

Comment:
@fingul remove the command renames and put the new information in the descriptions for the `--viewport-*` options instead of adding to the main command description